Update technical-stack.md (#4)
buerokratt/Buerokratt-onboarding: documents/technical-stack.md

## Turvis
_TO DO_

> ## Other  

Reviewer: ```suggestion
```

> Customer-service, is backend tool for administrating purposes to be used by customer service agent and admins  

Reviewer: ```suggestion
## Back-office GUI
A GUI-based tool for administrating purposes to be used by CSA and administrators
```

> Chat-widget, is frontend tool to be used for interaction between user and bot/customer service agent

Reviewer: ```suggestion
## Chat-widget
A frontend tool to be used for interaction between End User and Chatbot / CSA
```
